packages/site/src/utils/demo-buttons.js

```diff
@@ -5,86 +5,128 @@ import styled, { css } from 'react-emotion'
 import _GatsbyLink from 'gatsby-link'
 import { ErrorBoundary } from '../components/live'
 
-export const stringCode = `const Link = styled.a\`
-  display: inline-block;
-  border-radius: 5px;
-  padding: 16px;
+const stringLinks = `
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+
   margin: 1rem 0;
-  width: 12rem;
-  background: \${props =>
-    props.primary &&
-    'linear-gradient(90deg, #D26AC2, #46C9E5)'};
+
+  @media (min-width: 768px) {
+    flex-direction: row;
+    justify-content: flex-start;
+  }
+`
+
+const stringLink = `
+  box-sizing: border-box;
+  overflow: hidden;
+
+  min-width: 12rem;
+  margin: 0 auto 20px;
+  padding: 16px;
+  border-radius: 5px;
   text-decoration: none;
-  color: \${props =>
-    props.primary ? '#1D2029' : '#D26AC2'};
+  border: \${props => props.primary ? 'none' : '3px solid currentColor'};
+  background: \${props => props.primary && 'linear-gradient(90deg, #D26AC2, #46C9E5)'};
+  color: \${props => props.primary ? '#1D2029' : '#D26AC2'};
+
   &:hover {
     opacity: 0.95;
   }
-\``
 
-export const objectCode = `const Link = styled.a(props => ({
-  display: 'inline-block',
+  @media (min-width: 768px) {
+    margin: 0 20px 0 0;
+
+    &:last-child {
+      margin: 0;
+    }
+  }
+`
+
+export const stringCode = `
+const Links = styled.div\`${stringLinks}\`
+
+const Link = styled.a\`${stringLink}\`
+`
+
+export const objectCode = `
+const Links = styled.div(props => ({
+  display: 'flex',
+  flexDirection: 'column',
+  justifyContent: 'center',
+
+  margin: '1rem 0',
+
+  '@media (min-width: 768px)': {
+    flexDirection: row,
+    justifyContent: flex-start
+  }
+}))
+
+const Link = styled.a(props => ({
+  boxSizing: 'border-box',
+  overflow: 'hidden',
   borderRadius: 5,
   padding: 16,
   margin: '1rem 0',
   width: '12rem',
-  background:
-    props.primary &&
-    'linear-gradient(90deg, #D26AC2, #46C9E5)',
+  border: props.primary ? 'none' : '3px solid currentColor',
+  background: props.primary && 'linear-gradient(90deg, #D26AC2, #46C9E5)',
   color: props.primary ? '#1D2029' : '#D26AC2',
   textDecoration: 'none',
   '&:hover': {
     opacity: '0.95'
+  },
+  '@media (min-width: 768px)': {
+    margin: '0 20px 0 0',
+
+    '&:last-child': {
+      margin: 0
+    }
   }
 }))`
 
 export const GatsbyLink = ({ primary, ...props }: *) => (
   <_GatsbyLink {...props} />
 )
 
+export const PrecompiledLinks = styled.div`
+  ${stringLinks};
```

Comment:
This breaks the site when it's built since the function interpolations are escaped so they're just strings. Also I feel like we don't need to make `Links` editable since the main point of having them editable is to let people try the API and having two components to edit doesn't show more of the API and it makes the homepage more cluttered/most people won't be able to see all the code without scrolling.

Reply:
My bad! I should have waited to merge this.